scripts/split-e2e-tests.ts

```diff
@@ -307,9 +307,9 @@ function splitTests(
     const isPkg = newJobName.endsWith('_pkg');
     if (!isPkg) {
       (newJob.environment as any) = {
-        ...newJob.environment,
         AMPLIFY_DIR: '/home/circleci/repo/packages/amplify-cli/bin',
         AMPLIFY_PATH: '/home/circleci/repo/packages/amplify-cli/bin/amplify',
+        ...newJob.environment,
```

Comment:
what's the reason for switching this order?

Reply:
because if the `AMPLIFY_DIR` is specified in `config.base.json`, we shouldn't stomp on it.